content/docs/aws/deploy/install-kubeflow.md

```diff
@@ -99,27 +99,16 @@ Notes:
 Run kfctl to build your configuration files, so that you can edit the
 configuration before deploying Kubeflow:
 
-1. Run the `kfctl build` command to set up your configuration:
+1. Download manifest file
```

Comment:
Since there's only one step here, you can remove the numbered list.

Reply:
nice catch